topics/epigenetics/tutorials/ewas-suite/tutorial.md

```diff
@@ -86,31 +88,52 @@ The first step of the Infinium Human Methylation BeadChip array analysis is raw
 >    > https://zenodo.org/record/1251211/files/GSM1588706_8795207135_R02C02_Grn.idat
 >    > https://zenodo.org/record/1251211/files/GSM1588705_8795207119_R05C02_Grn.idat
 >    > https://zenodo.org/record/1251211/files/GSM1588707_8795207119_R06C02_Grn.idat
+>    > https://zenodo.org/record/1251211/files/phenotypeTable.txt
 >    > ```
 >    {: .details}
 >
 >    {% snippet faqs/galaxy/datasets_import_via_link.md %}
 >
 >    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>    > <comment-title>Phenotype table</comment-title>
+>    > Phenotype table can be in different sizes with different arguments, however the second column is required to contain phenotype covariate information for each sample.
+>    {: .comment}
+>
+> 3. Run **UCSC Main** {% icon tool %} to obtain the reference genome. The tool will take you to the **UCSC table browser**. Use the following parameters to extract the reference genome
+>   - *"clade"*: `Mammal`
+>   - *"genome"*: `Human`
+>   - *"assembly"*: `Feb. 2009 (GRCh37/hg19)`
+>   - *"group"*: `Regulation`
+>   - *"track"*: `HAIB Methyl450`
+>   - *"table"*: `GM12878 (wgEncodeHaibMethyl450Gm12878SitesRep1)`
+>   - *"region"*: `genome`
+>   - *"output format"*: `GTF - gene transfer (limited)`
+>   - *"Send output to"*: `Galaxy` (only)
+>   - Click on the **get output** button at the bottom of the screen
+>   - On the next page, click on the **Send Query to Galaxy** button
+>   - Wait for the upload to finish
 >
-> 3. Run **Infinium Human Methylation BeadChip** {% icon tool %} with the following parameters:
->    - {% icon param-files %} *"red channel files"*: all files ending in `_Red`
->    - {% icon param-files %} *"green channel files"*: all files ending in `Grn`
-> ![Raw_intensity_data_loading](../../images/1Raw_intensity_data_loading.png)
+> ![UCSC](../../images/5UCSC.png "UCSC")
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*

The figure caption here is also not terribly useful, you might choose to remove it or say something like "Screenshot of correct UCSC configuration"